index.js

```diff
@@ -65,23 +71,23 @@ exports.handler = function(event, context, callback) {
 /**
  * This function gets a tableID, finds the viewID, if one exists
  * and then grabs all the records in that view to be merged
- * @param {Number} tableId 
+ * @param {Number} tableId
  */
 function getRecordsThatNeedToBeMerged(tableId){
     //first figure out if we have a viewId associated
     //with this table Id
     var viewId = getViewForTable(tableId);
-    console.log("ViewId is: " + viewId);
+    log.log("ViewId is: " + viewId);
```

Comment:
Use the new templating syntax to be consistent with existing code. Just going to comment once; but please refactor throughout.